### ISSUE TITLE
Is it possible to fine-tune without Azure AI studio?

### ISSUE THREAD

**Issue by mark86v1** [ShishirPatil/gorilla#286]
Is it possible to implement RAFT without Azure AI studio? I am planning to use open-source models.

**Comment by Danielskry** [comment]
Yes, you can fine-tune or implement RAFT without Azure AI Studio, provided you have access to the appropriate hardware. One challenging aspect is determining whether the model you wish to fine-tune fits within the available vRAM memory (e.g., loading a model onto CUDA). I suggest using a calculator (can be found on Azure, Hugging Face, etc.) to assist with this process of selecting the proper model and GPU. Fine-tuning models can quickly become expensive and may exceed the capabilities of standard consumer GPUs, which is why many opt for platforms like Azure AI Studio for these tasks.

**Comment by songole** [comment]
is there a skypilot recipe to finetune this on any cloud?
